scalableminds/webknossos
# Fix that data was not rendered when zoomed in too much

The earlier fix caused a regression, because it didn't distinguish between having zoomed in so much that a viewport is covered by only one bucket and a viewport not being visible at all. Both resulted in a height and width of 0 in bucket coordinates. I introduced a new flag which checks the actual viewport width which is a float and, therefore, is only 0 if the viewport is not visible.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Zoom in as much as possible in a dataset -> Data should always be rendered
- I checked (via breakpoint) that the bucket picking is aborted if the 3D view is maximized (to confirm that https://github.com/scalableminds/webknossos/pull/5746 which was fixed with the code that caused the regression remains fixed)

### Issues:
- fixes #5755 

------
